browser/src/control/Control.PartsPreview.js

```diff
@@ -225,12 +244,20 @@ L.Control.PartsPreview = L.Control.extend({
 		}, this);
 
 		var that = this;
-		var pcw = document.getElementById('presentation-controls-wrapper');
+		L.DomEvent.on(frame, 'contextmenu', function(e) {
+			var nPos = undefined;
+			if (this.isPaddingClick(frame, e, 'top')) {
+				nPos = frame.id.replace(/^\D+/g, '');
```

Comment:
Here I tried to use frame id but i saw the id's are not secure to use. Because order can change easly. Then i trid to use i variable as ordered index number but didn't work. Any idea about how can understand where the mouspointer is clicked?

Reply:
we use "hash" to identify slides

Reply:
That was the old comment szymon and problem solved in time, Here slide hash's are not useful. I have to use slide index, Before slide indexes can be not in order when we move a slide. But currently problem is not present. Frame indexes give true position info us.